madebynota/portfolio
# Add OG embed image

So that when we share the link to the site online, we get a richer preview experience.

### Test Plan
* Run the server locally
* Create an ngrok tunner
* Paste the ngrok URL into the FB sharing debugger https://developers.facebook.com/tools/debug
* Ensure that the link preview looks good



<img width="911" alt="Screen Shot 2022-11-28 at 9 03 40 AM" src="https://user-images.githubusercontent.com/6589909/204296957-f9ba6866-9286-423b-b72c-8b1166c5c76f.png">
